OsmAnd/src/net/osmand/plus/base/FailSafeFunctions.java

```diff
@@ -167,6 +167,8 @@ protected void onPostExecute(GPXFile result) {
 	
 	public static void enterRoutingMode(@NonNull MapActivity mapActivity, @Nullable GPXRouteParamsBuilder gpxRoute) {
 		OsmandApplication app = mapActivity.getMyApplication();
+		app.logRoutingEvent("enterRoutingMode gpxRoute " + gpxRoute);
```

Comment:
gpxRoute has good toString?

Reply:
It should be enough - https://github.com/osmandapp/OsmAnd/pull/14770/files#diff-b54a0d5f4cefb955f45bb9466c923393fd3ea81298c5caca0d8bf5afe3898460R248